audio5.js

```diff
@@ -723,9 +733,8 @@
      */
     play: function () {
       if(this.audio) {
-        var playPromise = this.audio.play();
+        this.audio.play();
```

Comment:
I've spotted a difference generated from the build process.  This has come from the grunt task, can you advise how to avoid rebuilding causing this difference? I don't think this pull request can be accepted given this difference has been applied via the grunt tasks.